guide/src/SUMMARY.md

```diff
@@ -0,0 +1,23 @@
+# Summary
+
+[Yarner -- Introduction](./introduction.md)
+
+<!-- - [How is works](./how-it-works.md) -->
```

Comment:
Left over comment?

Reply:
The chapter is still there, but I am currently not sure if it is useful at all, or if the introductions explains it already. Also, If I include it, I am also not sure if it should be the first (numbered) chapter, or if it should be placed after "Getting started" in order to avoid hindrance for people to just jump into trying it out.

Reply:
From skimming both, I would suggest folding it into the introduction.

Reply:
I decided to remove it. Most of the content is already present in `Getting started`.